django_project/frontend/src/pages/Admin/index.jsx

```diff
@@ -52,12 +54,15 @@ export const pageNames = {
  * Base Admin App that contains side navigation and receive children
  * @param {string} pageName Current page name.
  * @param {React.Component} children React component to be rendered
+ * @param renderLeftSidebar
+ * @param props
  */
-export function AdminPage({ pageName, children, ...props }) {
+export function AdminPage({ pageName, children, renderLeftSidebar=true, ...props }) {
+  console.log(renderLeftSidebar)
   return (
     <App className='Admin' hideNavbar={true}>
-      <SideNavigation pageName={pageName}
-                      minified={props.minifySideNavigation}/>
+      { renderLeftSidebar ? <SideNavigation pageName={pageName}
```

Comment:
Hi @zamuzakki 
Not sure why we need this condition as we should always show side navigation

Reply:
@meomancer Both https://github.com/unicef-drp/GeoSight-OS/blob/d1dc1ece25a5d1fc2c24d593edb2752e3a4a6f7f/django_project/frontend/src/components/AdminList/index.tsx#L69 (one that calls Server Table) and MultipleAdminContent (which is used in tabbed admin page) ends up calling AdminPage, resulting in double left sidebar. So I add new prop to control left sidebar rendering.

Reply:
You don't need this, as you can use AdminListContent on Logs list
As my other comment